ynput/OpenPype
# Max: Add missing repair action in validate resolution setting

## Changelog Description
Add missing repair action for validate resolution setting

## Additional info
n/a

## Testing notes:
1. Launch Max via Ayon launcher
2. Create Render/Review instance
3. Enable validate resolution setting
4. If your scene has mismatched data with those in DB, it will error out.
5. The repair action should allow you to reset resolution setting according to the DB.
